ArduPlane/quadplane.cpp

```diff
@@ -1746,6 +1766,8 @@ void QuadPlane::vtol_position_controller(void)
     float ekfGndSpdLimit, ekfNavVelGainScaler;    
     ahrs.getEkfControlLimits(ekfGndSpdLimit, ekfNavVelGainScaler);
 
+    check_attitude_relax();
```

Comment:
Should this be done before the wp nav controller gets updated in setup_target_position()? (I haven't fully chased how that interacts with pos control and attitude control)

Reply:
This is actually inconsistently done. `vtol_position_controller` and `waypoint_controller` setup_target_position() first, while `takeoff_controller` relaxes then setups the target position.

Reply:
I don't think it matters, but I've changed it to be consistent anyway